ci/build-scripts/build-backend.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+echo "building backend"
+cd ./backend/
+
+source /opt/rust.env
+
+export DATABASE_URL=postgres://cidb:cidb@127.0.0.1/cidb
```

Comment:
What about setting this in ci/Jenkinsfile to keep the script more useful for devs? https://www.jenkins.io/doc/pipeline/tour/environment/